tests/msc2716_test.go

```diff
@@ -782,6 +782,9 @@ func TestImportHistoricalMessages(t *testing.T) {
 				eventIdBefore := eventIDsBefore[0]
 				timeAfterEventBefore := time.Now()
 
+				// Create eventIDsAfter to avoid the "No forward extremities left!" 500 error from Synapse
+				createMessagesInRoom(t, alice, roomID, 2)
```

Comment:
Poor comment wording. You reference `eventIDsAfter` like it's a variable but it isn't used here?

Do you really mean:

> Create *events after* to avoid the "No forward extremities left!" 500 error from Synapse

?

What does this have to do with room versions anyway? This just seems to be a logic error?